packages/components/src/components/icon/icon.custom.test.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  aTimeout, elementUpdated, expect, fixture, html,
+} from '@open-wc/testing';
+import { registerIconLibrary } from '../../../dist/synergy.js';
+import type SynIcon from './icon.js';
+
+describe('<syn-icon>', () => {
+  describe('svg sprite sheets', () => {
+    it('Should change size of icon even if "width" and "height" on an sprite svg was set', async () => {
+      registerIconLibrary('sprite', {
+        mutator: svg => svg.setAttribute('fill', 'currentColor'),
+        resolver: name => `/public/sprite.svg#${name}`,
+        spriteSheet: true,
+      });
+
+      const el = await fixture<SynIcon>(html`
+        <div style="font-size: 24px">
+          <syn-icon name="refresh" library="sprite"></syn-icon>
+        </div>
+      `);
+      const icon = el.querySelector('syn-icon')!;
+
+      await elementUpdated(icon);
+
+      const svg = icon.shadowRoot!.querySelector("svg[part='svg']")!;
+      const use = svg.querySelector("use[href='/public/sprite.svg#refresh']")!;
+
+      expect(svg).to.be.instanceof(SVGElement);
+      expect(use).to.be.instanceof(SVGUseElement);
+
+      // This is kind of hacky...but with no way to check "load", we just use a timeout
+      await aTimeout(1000);
```

Comment:
Don´t know why this was there at all, the test seems to run fine without this line (at least on my local system). Does not matter that much, just wanted to point it out.

Reply:
I took it from shoelace.
If not doing it, the test runs two times, because otherwise the first `width` and `height` are `0`.

Running tests with the timeout:

![image](https://github.com/user-attachments/assets/1078a3ce-5e00-4b0d-984c-058a69fdd4f6)

Running the tests without the timeout (sometimes it even seems to fail):
![image](https://github.com/user-attachments/assets/c95897a8-71bd-45e6-a157-2539221a4873)
![image](https://github.com/user-attachments/assets/800f2012-9a34-4d39-bf33-ab8a5b2bf0e9)